platform/darwin/src/CFHandle.hpp

```diff
@@ -0,0 +1,27 @@
+
```

Comment:
Missing `#pragma once`

We also typically make sure that header files include all of the symbols they reference. Instead of defining all of the aliases in the header file, we can continue to define them where we need them.

Reply:
:+1:

Maybe we don't need CFHandle at all? It seems like something we could replace with a portable "unique handle holder" or maybe just `std::unique_ptr` with custom deleters?